src/test/java/gov/usgs/wma/waterdata/PruneTimeSeriesTest.java

```diff
@@ -0,0 +1,49 @@
+package gov.usgs.wma.waterdata;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import static gov.usgs.wma.waterdata.PruneTimeSeries.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment= SpringBootTest.WebEnvironment.NONE)
+public class PruneTimeSeriesTest {
+
+	@MockBean
+	private TimeSeriesDao tsDao;
+	private PruneTimeSeries pruneTs;
+	private RequestObject request;
+
+	@BeforeEach
+	public void beforeEach() {
+		pruneTs = new PruneTimeSeries(tsDao);
+		request = new RequestObject();
+		request.setDate("2020-01-01 10:10:10");
```

Comment:
Probably better to set the date in the test since one uses it and one overrides it.

Reply:
Yes - thanks for catching.